.github/workflows/build-docs.yml

```diff
@@ -1,21 +1,31 @@
 name: Deploy Docs
-on:
-  push:
```

Comment:
I think we still want to build on landing to master... I believe this is important with how the docs push is run.

Why are we changing the path filter?